src/style-sheets/generic/_button.scss

```diff
@@ -1,32 +1,42 @@
 .button {
-    width: 189px;
-    height: 49px;
-    border-radius: 4px;
-    border-width: 0;
+  width: 189px;
+  height: 49px;
+  border-radius: 4px;
+  border-width: 0;
 
-    font-size: 1.25rem;
-    cursor: pointer;
+  font-size: 1.25rem;
+  cursor: pointer;
 }
 
 .primary {
-    background-color: $olLTAccent;
-    color: $white;
-    font-family: $olBaseFont;
+  background-color: $olLTAccent;
+  color: $white;
+  font-family: $olBaseFont;
+}
+
+.primary:hover {
+  background-color: #259333;
+}
+
+.primary:active {
+  background-color: #f4fbf5;
+  color: #43a84f;
+  border-radius: 4px;
 }
 
 .disabled {
-    background-color: $olLTDisabled;
```

Comment:
is this linting being done by your local ide?

Reply:
Hey man, thanks for the comprehensive review. 
By linting do you mean the text formatting? I haven't heard that term before. It may be my prettier extension in vs code. I thought I had it disabled tho, apologies.

Reply:
Oh you're fine, I was just wondering if our formatting hook was doing that. And yes text formatting aka linting. Prettier is called a linter